tests/unit/core.js

```diff
@@ -872,6 +877,7 @@ exports.testConstModification = function (test) {
       .addError(8, "Attempting to override 'a' which is a constant.")
       .addError(8, "You might be leaking a variable (a) here.")
       .addError(53, "Missing '()' invoking a constructor.")
+      // .addError(17, "Attempting to override 'f' which is a constant.") TODO
```

Comment:
line 17 is `  let x = -a;` though, so I'm not too sure why this error is here or commented out